### PR TITLE
[Story] Expand sdk with surveystate

### PR DESCRIPTION
**Pull request by hnuguse**
https://github.com/NIPOSoftwareBV/nfield/issues/1102

### CODE REVIEW

File: Library/Models/SurveyState.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ namespace Nfield.Models
     /// <summary>
     /// The fieldwork status of the survey
     /// </summary>
-    public enum SurveyStatus
+    public enum SurveyState
```
Comment (joukevandermaas) [inline Library/Models/SurveyState.cs]: You can't do this, this is a breaking change.

Comment (carloscoceraperez) [reply Library/Models/SurveyState.cs]: What is it breaking? Its an internal model and all it's usages have been updated. At the end it's parsed into the same that SurveyStatus

Comment (carloscoceraperez) [reply Library/Models/SurveyState.cs]: You mean someone which was expecting a Survey.Status in the response and now that code it broken?

Comment (joukevandermaas) [reply Library/Models/SurveyState.cs]: it's public, not internal. so someone could have used it

**Pull request closed without merging**